guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -404,7 +404,7 @@ Sets the declared block or container element as a child of another element in th
    </tbody>
 </table>
 
-### &lt;remove&gt; {#fedg_layout_xml-instruc_ex_rmv}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.